### DISCUSSION TITLE
assignment vs project

### DISCUSSION THREAD

**Discussion by axecopfire** [Vets-Who-Code/course-development-guide#5] [General]
What are the Sign-off sheet Standards, are they assignments or projects. Projects are more open ended where assignments are more direct and just cover the lessons? Are they both, what should the scope of it be?

**Comment by myquite** [comment]
I see assignments as opportunities for the learner to practice specific tasks to a well-defined standard that encourages rapid and detailed feedback. For assignments, the feedback should be clear and direct. 

On the other hand, projects are opportunities for learners to apply the skills they're developing during assignments to their interests. The feedback during projects is less direct and more inquiry-based, where the goal is to guide the project to completion.

**Reply by axecopfire** [reply]
Can you provide examples for when you would use an assignment vs when you would use a project? At the end of a course how many projects vs how many assignments should be done?

**Reply by myquite** [reply]
I view the task, conditions, and standards we've written as assignments. I see projects as a parallel track.

**Reply by axecopfire** [reply]
Seems like having students balancing two different pieces of parallel work seems like a specific challenge. I don't think Junior devs are really expected to do that at work.

**Reply by myquite** [reply]
There are already parallel tracks. They are learning new skills and trying to get a job. A Project track is a more formalized approach to the latter.

**Reply by axecopfire** [reply]
IMO We should have a goal of reducing parallelism as much as possible, so we can focus on quality and distill value in a single experience. If we want to have a Project creation track (like software CEO) make that a separate qualification. If we want to have a job search qualification, lets make that a separate qualification. I think we shouldn't assume parallelism and multiple concerns, that could be up to each student whether they want to parallelize or not.

**Reply by myquite** [reply]
OK. We'll design with that goal in mind.
